### ISSUE TITLE
Links to mentions in notifications page

### ISSUE THREAD

**Issue by adamralph** [JabbR/JabbR#1053]
In the notifications page, it would great to have a link to the mention in the chat room so it's possible to see the context in which the mention was made.

Currently, if the mention was some time ago, it can take a long time to navigate back to that point in time.

**Comment by hnrkndrssn** [comment]
I agree that would be beneficial to us all, but would only work if the message is in the list of messages in the room...as we currently truncate the message history of a room (I believe) the message might have been removed, or if the mention was while you didn't have jabbr.net running, it could also be that the message isn't in the message list of the room...just something we would have to think about...

---

From: Adam Ralph
Sent: 14/01/2014 9:05
To: JabbR/JabbR
Subject: [JabbR] Links to mentions in notifications page (#1053)

In the notifications page, it would great to have a link to the mention in the chat room so it's possible to see the context in which the mentions was made.

Currently, if the mention was some time ago, it can take a long time to navigate back to that point in time.

---

Reply to this email directly or view it on GitHub:
https://github.com/JabbR/JabbR/issues/1053

**Comment by adamralph** [comment]
If the message has been cleared out of the database that's just too bad. I can always download the room history and trawl through that (or if it's even gone from that then just too bad).

At the moment, I go to the room, and then scroll up hitting many 'Loading previous messages...' until I get to the point I want to see. All I'm really looking for is an automation of that process, which would preferably load all the messages back to that point in history in one hit rather than many.
